src/sessions/SessionManager.ts

```diff
@@ -137,7 +137,7 @@ export class SessionManager {
     public addSessionAttributes(sessionAttributes: {
         [k: string]: string | number | boolean;
     }) {
-        this.attributes = { ...this.attributes, ...sessionAttributes };
+        this.attributes = { ...sessionAttributes, ...this.attributes };
```

Comment:
I'm still skeptical about not letting applications overwrite the built-in metadata.

While not permitting overwrites might prevent users from accidentally overwriting built-in metadata, there may be cases where we want to overwrite the built-in metadata.

Likelihood of overwriting accidents  vs likelihood of use cases overwriting.

One example is if one wanted to spoof the domain, since RUM doesn't currently support multiple domains in the app monitor config.

Reply:
We will apply this change as-is, which is to disallow overwriting built-in metadata. This is under the assumption that we can change the behavior to be more permissive in the future.